upstream/suite_test.go

```diff
@@ -0,0 +1,113 @@
+package upstream
```

Comment:
Can you move this file to the root ?

I also did something similar with rbac, if we use this in root the db setup can be used by all the tests

Reply:
Make sense. Although a slight problem would be how the packages (`upstream` & `rbac`) would access the db objects. This is unlike duty where all the tests are written in the same package.

Reply:
We can experiment with importing